client/src/components/SelectionCard.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+import { Card } from 'antd';
+
+const { Meta } = Card;
+
+const SelectionCard = ({ coverImage, selectionTitle, routePath }) => {
+  const history = useHistory();
+  
+  const bodyStyle = {
```

Comment:
I think it would be better to save our styling in an external CSS stylesheet and reference it here as className (I believe that’s better practice)

Reply:
Yea I was going to externalize the CSS for this at first but then ant design actually has props for passing in the CSS properties props(hence me doing it here). Otherwise we'd have to override the card-body CSS from ant design (whatever that css may be)